src/index.ts

```diff
@@ -1 +1,32 @@
-console.log("Hello");
+import { Pool, PoolClient, QueryResult } from 'pg';
+import { config } from 'dotenv';
+
+config(); // Load environment variables from .env file
+
+const pool = new Pool({
+  host: process.env.DB_HOST,
+  port: parseInt(process.env.DB_PORT || '5432', 10),
+  database: process.env.DB_NAME,
+  user: process.env.DB_USER,
+  password: process.env.DB_PASSWORD,
+});
+
+async function fetchData(): Promise<void> {
+    let client: PoolClient | undefined;
+  
+    try {
+      client = await pool.connect();
+      try {
+        const res: QueryResult = await client.query('SELECT * FROM "users"');
+        console.log('Connected to PostgreSQL:', res.rows);
```

Comment:
why are we fetching data here ? or its just a test ?

Reply:
yes it was just to test